sources/packages/backend/apps/db-migrations/src/sql/CASDistributionAccounts/Create-cas-distribution-accounts.sql

```diff
@@ -0,0 +1,44 @@
+CREATE TABLE sims.cas_distribution_accounts(
+  id SERIAL PRIMARY KEY,
+  award_value_code VARCHAR(10) NOT NULL,
+  offering_intensity sims.offering_intensity NOT NULL,
+  operation_code CHAR(2) NOT NULL,
+  distribution_account VARCHAR(40) NOT NULL,
```

Comment:
I see that this column is not mentioned in technical ACs. So is it added to specify the particular distribution account name/code ?

Reply:
It was not mentioned in the technical AC(I can add) but it represents the below business AC
```
Distribution Accounts per DR/CR per Award
```
Also in the spreadsheet provided by the business, as below.
![image](https://github.com/user-attachments/assets/70220647-e75f-43d9-b43e-d8827ac86f70)